src/Microsoft.ML.StandardLearners/Standard/Online/OnlineLinear.cs

```diff
@@ -19,27 +19,41 @@ namespace Microsoft.ML.Trainers.Online
 
     public abstract class OnlineLinearArguments : LearnerInputBaseWithLabel
     {
+        /// <summary>
+        /// Number of training iterations through the data.
```

Comment:
>Number of training iterations [](start = 12, length = 29)

Number of passes through the dataset.

Reply:
done

---
In reply to: [256223998](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256223998) [](ancestors = 256223998)